Automattic/woocommerce-payments
# Remove WooPay subscriptions user check endpoint

#### Changes proposed in this Pull Request









This PR removes the endpoint used to check if a guest user buying a subscription has a merchant site account and ask them to login before redirect to WooPay, we do not support this flow (guest user +  WooPay subscription) anymore since https://github.com/Automattic/woocommerce-payments/issues/6131 so there's no need for this endpoint.

#### Testing instructions





As this endpoint is already not being called on `trunk`, just do a normal WooPay checkout test to ensure everything is working.

* With WooPay enable and a guest user on merchant site, add a non subscription product to the cart.
* Access checkout page and type a valid WooPay, the modal should appear.
* Add a subscription product to the cart.
* Access checkout page and type a valid WooPay, the modal should not appear.

-------------------


**Post merge**
